play/src/components/ScoreIndicator.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState, useEffect } from 'react';
+import { makeStyles, ThemeProvider} from '@mui/styles';
+import { Typography } from "@mui/material";
+import { isNullOrUndefined } from '@righton/networking';
+
+interface ScoreIndicatorProps {
+  newPoints: number;
+  score: number | null;
```

Comment:
Any reason we go for nullable score? We should try to clean these ones up as well and have them defaulted to 0 always to reduce null checks and increase code readability

Reply:
yeah, totally agreed (see comment in PR580, cardanswers). An issue has been added to github for this as I'd like to handle it in a subsequent pr (and kill off some UI stuff first)